recipe/meta.yaml

```diff
@@ -7,47 +7,43 @@ package:
-      - patches/0006-Add-absl_log-systemlib.patch
-      - patches/0007-Omit-linking-to-layout_proto_cc-if-protobuf-linkage-.patch
-      - patches/0008-Fix-further-abseil-linkage.patch
-      - patches/0009-Add-constraint-to-pybind11-systemlib.patch
-      - patches/0010-Different-file-ending-for-flatbuffers-LICENSE.patch
-      - patches/0011-Use-correct-hermetic-python.patch
-      - patches/0012-Add-well_known_types_py_pb2-to-protobuf-systemlib.patch
-      - patches/0013-Add-protobuf-toolchain.patch
-      - patches/0014-fix-genproto.patch
-      - patches/0015-Remove-some-usage-of-absl-str_format-in-CUDA.patch  # [(cuda_compiler_version or "").startswith("11")]
-      - patches/0016-Patch-ml_dtype-to-make-constructor-unambiguous.patch
-      - patches/0017-Load-cuda_build_defs-from-tsl.patch
-      - patches/0018-Use-PYTHON-for-build-wheel.patch
-      - patches/0019-Always-use-Linux-sed-style.patch
-      - patches/0020-Adjust-relative-path-for-libdevice.patch  # [(cuda_compiler_version or "").startswith("12")]
+      - patches/0006-Omit-linking-to-layout_proto_cc-if-protobuf-linkage-.patch
+      - patches/0007-Fix-further-abseil-linkage.patch
+      - patches/0008-Add-constraint-to-pybind11-systemlib.patch
+      - patches/0009-Different-file-ending-for-flatbuffers-LICENSE.patch
+      - patches/0010-Use-correct-hermetic-python.patch
+      - patches/0011-Add-well_known_types_py_pb2-to-protobuf-systemlib.patch
+      - patches/0012-Add-protobuf-toolchain.patch
+      - patches/0013-fix-genproto.patch
+      - patches/0014-Remove-some-usage-of-absl-str_format-in-CUDA.patch
+      - patches/0015-Patch-ml_dtype-to-make-constructor-unambiguous.patch
+      - patches/0016-Use-PYTHON-for-build-wheel.patch
+      - patches/0017-Always-use-linux-sed-style.patch
+      - patches/0018-Adjust-relative-path-for-libdevice.patch
+      - patches/0019-Update-flatbuffers-static-assert.patch
       # https://github.com/tensorflow/tensorflow/pull/62684
       - patches/62684.patch
+
+      # Not ported yet
+      # - patches/0023-Fix-missing-includes-needed-for-Abseil-lts_2024_01_1.patch
+      # - patches/0024-Link-to-absl_log_flags-instead-of-absl_flags.patch
+      # - patches/0025-Update-ABSL-Log-Definition-for-libabsl_vlog_config_i.patch
   - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
     sha256: 2d7e100b1878084da34b5e23b49a0cbb5ee8a7add74b7dd189a82ada1cf85530
     folder: tensorflow-estimator
-  # yes, the headers come from a different version than the python package required below.
-  - url: https://raw.githubusercontent.com/jax-ml/ml_dtypes/v0.3.1/ml_dtypes/include/float8.h
-    fn: float8.h
-    sha256: d2798fad4e64375b566b1df1d7bc440313e4b1024ca08f12cead3eaa4b73ff72
-  - url: https://raw.githubusercontent.com/jax-ml/ml_dtypes/v0.3.1/ml_dtypes/include/int4.h
-    fn: int4.h
-    sha256: b3a9970c3c6b169c41ac2fd4375f668d3fd1b492d48b912d89415fa1522a8f50
```

Comment:
Since there's no comment that tells me _why_ we need the vendored headers, I'm going to assume that `ml_types 0.3.2` is going to be an adequate replacement.

Reply:
See also discussion [previously](https://github.com/conda-forge/tensorflow-feedstock/pull/353/files#r1407419483) and the new [`libml_dtypes-headers`](https://github.com/conda-forge/libml_dtypes-headers-feedstock). My understanding is that upstream is back in alignment and we don't need to do this manually anymore, but would appreciate your input @xhochy.

Reply:
FWIW, I've added `libml_dtypes-headers` as a run-time dependence of `libtensorflow_cc`, which is enough to make the current tests pass.